src/System.Private.CoreLib/shared/System/Type.Enum.cs

```diff
@@ -169,7 +170,7 @@ private static int BinarySearch(Array array, object value)
             return Array.BinarySearch(ulArray, ulValue);
         }
 
-        internal static bool IsIntegerType(Type t)
+        internal static bool IsIntegerType(Type? t)
```

Comment:
I don't think this can be nullable, maybe one of the cases where the tool over annotated? It doesn't make sense to pass down a null type to this API. cc @stephentoub

Reply:
i think it was annotated by tool and i didn't had any reason to remove that, but agree with you, making it nullable doesn't make sense, removing nullable